packages/vue/src/components/range/RangeInput.jsx

```diff
@@ -113,7 +113,7 @@ const RangeInput = {
 			}
 		},
 		handleOnChange(value) {
-			this.handleChange(value, 'change');
+			this.handleChange(value ?? this.$props.range, 'change');			
```

Comment:
@mohdashraf010897 Avoid using `Nullish` operator, it is not supported by some browsers.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
fixed issues https://github.com/appbaseio/reactivesearch/pull/1853/commits/6a1e2a831f23dbf88a79eed5f676751d1909daea